endpoints.json

```diff
@@ -30,5 +30,99 @@
+    "description": "serves an object with the review by the given review_id",
+    "queries": [],
+    "exampleResponse": {
+      "reviews": {
+        "category": "euro game",
+        "created_at": "2021-01-18T10:00:20.514Z",
+        "designer": "Uwe Rosenberg",
+        "owner": "mallionaire",
+        "review_body": "Farmyard fun!",
+        "review_id": 1,
+        "review_img_url":
+          "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+        "title": "Agricola",
+        "votes": 1,
+        "comment_count": 0
+      }
+    }
+  },
+
+  "PATCH /api/reviews/:review_id": {
+    "description": "updates the review with review_id by incrementing the votes by the number given in the parameter inc_votes from the body. Returns the updated review",
+    "queries": [],
+    "exampleBody": {"inc_votes": 1},
+    "exampleResponse": {
+      "reviews": {
+        "title": "Agricola",
+        "designer": "Uwe Rosenberg",
+        "owner": "mallionaire",
+        "review_img_url":
+          "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+        "review_body": "Farmyard fun!",
+        "category": "euro game",
+        "created_at": "2021-01-18T10:00:20.514Z",
+        "votes": 2
+      }
+    }
+  },
+
+  "GET /api/review/:review_id/comments": {
+    "description": "serves an array of objects with the comments of the review with review_id given",
+    "queries": [],
+    "exampleResponse": [
+      {
+        "comment_id": 6,
+        "body": "Not sure about dogs, but my cat likes to get involved with board games, the boxes are their particular favourite",
+        "review_id": 3,
+        "author": "philippaclaire9",
+        "votes": 10,
+        "created_at": "2021-03-27T19:49:48.110Z"
+      },
+      {
+        "comment_id": 3,
+        "body": "I didn't know dogs could play games",
+        "review_id": 3,
+        "author": "philippaclaire9",
+        "votes": 10,
+        "created_at": "2021-01-18T10:09:48.110Z"
+      },
+      {
+        "comment_id": 2,
+        "body": "My dog loved this game too!",
+        "review_id": 3,
+        "author": "mallionaire",
+        "votes": 13,
+        "created_at": "2021-01-18T10:09:05.410Z"
+      }
+    ]
+  },
+
+  "POST /api/review/:review_id/comments": {
+    "description": "adds a comment to the database about the review of review_id and responds with the added comment information",
+    "queries": [],
+    "exampleResponse": {
+      "comment_id": 7,
+      "review_id": 1,
+      "votes": 0,
+      "created_at": "2022-11-16T12:17:11.065Z",
+      "author": "bainesface",
+      "body": "EPIC board game!"
+    }
+  },
+
+  "GET /api/users": {
+    "description": "serves an array of all the users",
+    "queries": [],
+    "exampleResponse": [
+      {
+        "username": "mallionaire",
+        "name": "haz",
+        "avatar_url": "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg"
+      }
+    ]
```

Comment:
You should also include `/api/reviews` This one should also mention the sorting and filtering functionality